index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700;800;900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    <title>my mobile porifolio</title>
+</head>
+<body>
+    <header class="header-container fixed-top">
+            <div class="logo"><img src="/img/My Logo.png" alt="my logo"></div>
```

Comment:
🙂  Just like the former code reviewer suggested, you should use an anchor tag `a` for the logo due to convention, semantics, and user experience reasons.